flink-contrib/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBKeyedStateBackend.java

```diff
@@ -270,6 +277,8 @@ public RocksDBKeyedStateBackend(
 			new IncrementalSnapshotStrategy() :
 			new FullSnapshotStrategy();
 
+		this.writeOptions = new WriteOptions().setDisableWAL(false);
```

Comment:
We enable WAL in contrast to the previous state. This is intended, right?

Reply:
No, this was unintended...the double negation got me here.